torch/csrc/api/src/nn/init.cpp

```diff
@@ -146,6 +165,33 @@ Tensor uniform_(Tensor tensor, double low, double high) {
   return tensor.uniform_(low, high);
 }
 
+Tensor kaiming_uniform_(Tensor tensor, double a, FanMode mode, Nonlinearity nonlinearity) {
+  Fan fan(tensor);
+  const auto gain = calculate_gain(nonlinearity, a);
+  double std = 0.0;
+  if(mode == FanMode::fan_in) {
```

Comment:
Please also use `clang-format` on this file. The beauty factor is a bit off here